R/scale-type.R

```diff
@@ -25,7 +24,13 @@ find_scale <- function(aes, x, env = parent.frame()) {
 # Look for object first in parent environment and if not found, then in
 # ggplot2 namespace environment.  This makes it possible to override default
 # scales by setting them in the parent environment.
-find_global <- function(name, env, mode = "any") {
+find_global <- function(name, env, mode = "any", theme = ggplot2::theme()) {
+  ## there is presumably a more elegant way to test presence and mode in a list.
```

Comment:
Not sure what you mean. The code looks fine as is.

Reply:
Comment is related to a previous commit and should be ignored